src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx

```diff
@@ -4,21 +4,59 @@
  */
 
 import { User, WorkPackageStage } from 'shared';
+import { useState } from 'react';
 import { fullNamePipe } from '../../../utils/pipes';
 import PageBlock from '../../../layouts/PageBlock';
 import { FormControl, FormLabel, Grid, MenuItem, TextField } from '@mui/material';
 import ReactHookTextField from '../../../components/ReactHookTextField';
-import { Control, Controller, FieldErrorsImpl } from 'react-hook-form';
+import { Controller } from 'react-hook-form';
 import { DatePicker } from '@mui/x-date-pickers';
-import { WorkPackageEditFormPayload } from './WorkPackageEditContainer';
+import NERAutocomplete from '../../../components/NERAutocomplete';
 
 interface Props {
-  users: User[];
-  control: Control<WorkPackageEditFormPayload>;
-  errors: Partial<FieldErrorsImpl<WorkPackageEditFormPayload>>;
+  usersForPL: User[];
+  usersForPM: User[];
+  control: any;
+  errors: any;
 }
 
-const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) => {
+const WorkPackageEditDetails: React.FC<Props> = ({ usersForPL, usersForPM, control, errors }) => {
+  const [lead, setLead] = useState<User | null>(null);
```

Comment:
the default values should be the project's existing lead and manager (or null)


also how do these get incorporated into the submit? i don't think they do which is probably why this doesn't work